fastlane/lib/fastlane/actions/ensure_env_vars.rb

```diff
@@ -3,15 +3,12 @@ module Actions
     class EnsureEnvVarsAction < Action
       def self.run(params)
         variables = params[:env_vars]
+        missing_variables = variables.select { |variable| ENV[variable].to_s.strip.empty? }
 
-        variables.each do |variable|
-          next unless ENV[variable].to_s.strip.empty?
-
-          UI.user_error!("Missing environment variable '#{variable}'")
-        end
+        is_one = missing_variables.length == 1
+        UI.user_error!("Missing environment variable#{is_one ? '' : 's'} '#{missing_variables.join('\', \'')}'") unless missing_variables.empty?
```

Comment:
```suggestion
        UI.user_error!("Missing environment variable(s) '#{missing_variables.join('\', \'')}'") unless missing_variables.empty?
```

I understand that this logic comes from the below line but isn't this enough here? 🙂 (and reusing the same local variable for a different purpose doesn't sound good in general).

Reply:
This small logic `{is_one ? '' : 's'}` was breaking 2 existing unit tests, did not write this way in the first place. 
But, I agree with your suggestion, so applied it and updated the unit-tests too 🙇